tests-legacy/PrestaShopBundle/Utils/DatabaseCreator.php

```diff
@@ -47,7 +47,11 @@ public static function createTestDB()
         $install = new Install();
         \DbPDOCore::createDatabase(_DB_SERVER_, _DB_USER_, _DB_PASSWD_, _DB_NAME_, false);
         $install->clearDatabase(false);
-        $install->installDatabase(true);
+        if (!$install->installDatabase(true)) {
+            // Something went wrong during installation
+            exit(1);
```

Comment:
Can you throw an exception rather than using `exit` ? Too many unexpected behaviour happens from the use of exit.. (stoping tests suite, breaking ajax request, debug sessions, ruining friday nights...)

Reply:
We are in CLI context, the installDatabase already display an error message.